<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>851</width>
    <height>602</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>IMDG units data filler for MACS3</string>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonOpen">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>100</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Fill in IMDG units</string>
    </property>
   </widget>
   <widget class="QLabel" name="CXMLFileNameLable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>163</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>IMDG units file :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonBrowse">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>20</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLabel" name="manifestFileLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>IMDG Manifest file :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cxmlFileLine">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>381</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="manifestFileLine">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>381</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonBrowse_2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>60</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Save in cxml file to open in MACS3&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Save in file :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="newCXMLFileLine">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>381</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="totalIMDGUnitsLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Save in cxml file to open in MACS3&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Total IMDG units for export to MAC3:</string>
    </property>
   </widget>
   <widget class="QLabel" name="totalContainersLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Save in cxml file to open in MACS3&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Total containers handled:</string>
    </property>
   </widget>
   <widget class="QLabel" name="totalIMDGLabel">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>150</y>
      <width>45</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="totalContLabel">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>180</y>
      <width>45</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>220</y>
      <width>211</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="containerList">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>201</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>220</y>
      <width>241</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>520</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>851</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuContainer_Editor">
    <property name="title">
     <string>Container Editor</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionIMDGUnitHandler"/>
    <addaction name="separator"/>
    <addaction name="closeProgram"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionHow_to_use_it"/>
    <addaction name="actionExamples_of_files"/>
   </widget>
   <addaction name="menuContainer_Editor"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionlabel_one">
   <property name="text">
    <string>label one</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;label action&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionIMDGUnitHandler">
   <property name="text">
    <string>IMDG Unit Handler</string>
   </property>
  </action>
  <action name="closeProgram">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This software is intended for the creation&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionHow_ti_use_it">
   <property name="text">
    <string>How ti use it</string>
   </property>
  </action>
  <action name="actionHow_to_use_it">
   <property name="text">
    <string>How to use it</string>
   </property>
  </action>
  <action name="actionMACS3_link">
   <property name="text">
    <string>MACS3 link</string>
   </property>
  </action>
  <action name="actionExamples_of_files">
   <property name="text">
    <string>Examples of files</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
